website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -83,7 +83,15 @@ Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
 Credentials" page to finish configuring dbt Cloud to authenticate with Snowflake
 using a key pair.
 
-**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud -- you **must** add the passphrase that will be used to decrypt the and will receive an error if the `PRIVATE KEY PASSPHRASE` field is empty.
+:::**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
+:::
+
+
+**Example:**
+```sql
```

Comment:
Will want to close this ```sql out with an ending ```!  Currently the rest of the page is being pulled into this SQL block.

Reply:
Fixed this